src/_guides/whats-new.md

```diff
@@ -22,11 +22,29 @@ and follow the [Dart blog][].
 This section lists notable changes made from May 12, 2022,
 through August 30, 2022.
 For details about the 2.18 release,
-see [Dart 2.18: ][], and the [change log][https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180].
+see [Dart 2.18: Objective-C & Swift interop][],
+and the [change log][].
+
+[Dart 2.18: Objective-C & Swift interop]: https://medium.com/dartlang/dart-2-18-f4b3101f146c
+[change log]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180
```

Comment:
I think we generally don't put a space in "changelog" and perhaps we should be more specific what changelog this is referring to:

```suggestion
and the [SDK changelog][].

[Dart 2.18: Objective-C & Swift interop]: https://medium.com/dartlang/dart-2-18-f4b3101f146c
[SDK changelog]: https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#2180
```